### DISCUSSION TITLE
RFID card to download then play a sound?

### DISCUSSION THREAD

**Discussion by virginiebjt** [MiczFlor/RPi-Jukebox-RFID#2213] [Q&A]
Hey guys,
I'm trying to build my grand ma a simple jukebox where she could listen to some podcasts I downloaded for her and stored locally. But I would also like to send her new stuff once she has the box, that is to say, I'd like to mail her a RFID card, and then once she taps it on the box, it would fetch the mp3 sound (that i'd store on my website for example), then download it, save it locally, and then she'd be able to listen to it whenever she wants, whether she has wifi or not. Would that be possible? Thank you!

**Comment by AlvinSchiller** [comment]
Hey,
from what i am aware of is this feature is the closest to get what you describe: [Sychnronising-Phonieboxes-in-a-local-network](https://github.com/MiczFlor/RPi-Jukebox-RFID/wiki/Sychnronising-Phonieboxes-in-a-local-network).
This handles the filestransfer and card check pretty simliar to your describtion. Only things its based for a local network.
But if your server is also accessable with ssh it could work.

Or another idea would be to connect the phoniebox via VPN to your network. Or the other way around and you get a VPN access to your grandmas network (some modern routers have a VPN server build in). This way you could access it directly (samba, sftp) and upload new configurations and files.

**Comment by TomK32** [comment]
I want to implement the same, play/download files or podcast feeds.

In your case she'd still have to register the card on her phoniebox before she can use it. It'd be terrific to store that as a QR code and have a camera function on the webUI. Store the function and variables in the QR code for easier usage.

**Reply by AlvinSchiller** [reply]
Not quite correct :) 
The feature [Sychnronising-Phonieboxes-in-a-local-network](https://github.com/MiczFlor/RPi-Jukebox-RFID/wiki/Sychnronising-Phonieboxes-in-a-local-network) also syncs the card database. So you can add / change cards assignments in that file with an text editor on the server and on swiping cards the entries are synced and new files are downloaded. So you can manage the cards and files completely on the server side.
You only need to know the IDs of your cards beforehand to add them.

Edit: "Card Database" is for v3, but the mechanism is also available in v2 via the shortcut files.
